bundles/org.openhab.binding.mybmw/README.md

```diff
@@ -108,7 +108,7 @@ Same configuration is needed for all things
 | Parameter       | Type    | Description                           |
 |-----------------|---------|---------------------------------------|
 | vin             | text    | Vehicle Identification Number (VIN)   |
-| refreshInterval | integer | Refresh Interval in Minutes           |
+| refreshInterval | integer | Refresh Interval in Minutes (set to 0 if no automated refresh should be triggered)          |
```

Comment:
Can you fix the markdown? Maybe also list the default for refreshInterval (maybe little bit beyond the scope of this PR, but it would be usefull.)

Reply:
do you mean the line length? Fixed